pkg/config/config.go

```diff
@@ -23,8 +23,14 @@ type GitHubRepo struct {
 	Actions GitHubRepoActions `yaml:"actions"`
 }
 
+type GitHubOrgActions struct {
```

Comment:
Use different type for `GithubOrgActions` and `GithubRepoActions` so that they can be receivers of different methods.